### DISCUSSION TITLE
azure - expires before it finish updating container

### DISCUSSION THREAD

**Discussion by markat1** [microsoft/aspire#5858] [Q&A] [answered]
I'm trying to deploy my aspire project, but I'm running into a issue doing deployment.

I had one of my containers updating for a really long time, but it unfortunately ends up expiring in the end.

Is there a way I can extend the amount of time it should take updating a container?

![image](https://github.com/user-attachments/assets/fedc1676-b405-40d4-ba0d-7db37c02ebaa)

**Comment by davidfowl** [comment]
Does this happen all the time?

**Reply by markat1** [reply]
@davidfowl yeah - tried a couple of times  - same result - seems like it's  able to create container, but then tries to update it for a long time (maybe 30-40 min?) and ends up with "operation expired"

**Reply by markat1** [reply]
I'm missing some debug tools, because I can't really see what's causing it to take so long and then end up expiring. 

Sometimes I'm lucky that an error appears after a while, but not in this particular example.

**Reply by markat1** [reply, marked as answer]
I tried a lot of removing unnecessary code for my specific container and it finally worked!
